LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/IoTHubImpl/IoTHubRegistryManager.cs

```diff
@@ -62,5 +62,52 @@ public async Task<IDeviceTwin> UpdateTwinAsync(string deviceId, string moduleId,
 
         public Task RemoveDeviceAsync(string deviceId)
             => this.instance.RemoveDeviceAsync(deviceId);
+
+        public async Task<IEnumerable<IDeviceTwin>> GetEdgeDevicesAsync(CancellationToken token)
+        {
+            var q = this.instance.CreateQuery($"SELECT * FROM devices.modules where moduleId = '{LoRaToolsConstants.NetworkServerModuleId}'");
```

Comment:
Why differentiate GetEdgeDevicesAsync by returning the IEnumerable<IDeviceTwin> instead of a IRegistryPageResult<IDeviceTwin> like other methods added?